tools/azure-sdk-tools/devtools_testutils/proxy_startup.py

```diff
@@ -303,146 +304,34 @@ def set_common_sanitizers() -> None:
     batch_sanitizers[Sanitizer.REMOVE_HEADER] = [{"headers": headers_to_ignore}]
 
     # Remove OAuth interactions, which can contain client secrets and aren't necessary for playback testing
-    batch_sanitizers[Sanitizer.OAUTH_RESPONSE] = [None]
+    # TODO: Determine why this breaks some test playbacks. Since sensitive info in OAuth responses is sanitized
```

Comment:
I would imagine that test failures caused by the OAuth sanitizer come from tests not using a fake credential in playback. For example, `AzureRecordedTestCase.get_credential` (the preferred method of getting a test credential) returns a fake credential during playback that has a dummy `get_token` implementation so that it doesn't actually make requests. If a real credential is used, token requests will be made in playback and conflict with the OAuth sanitizing.

Reply:
To close the loop on this: you're right that the sensitive information is now being sanitized (central OAuth sanitizing was added as an immediate stopgap before robust central sanitizers), so we don't need to block on this. We'll just want to investigate why OAuth interaction behavior has changed in playback and try to restore this sanitizing when possible.